### PR TITLE
MaxQuant scores added 

### PR DESCRIPTION
**Pull request by ypriverol**
This PR includes a set of scores for MaxQuant-DIA experiments. In addition, we added a NIST msp reference to spectra

### CODE REVIEW

**Comment by ypriverol** [comment]
@edeutsch @mwalzer or anyone can review this PR @HUPO-PSI/psi-ms-cv

**Comment by ypriverol** [comment]
Agreed. Actually I tried to keep the trend and "standard" but if you don't like it, I will remove it. Because I don't like it either.

**Comment by ypriverol** [comment]
@edeutsch i have updated the terms

**Comment by ypriverol** [comment]
@edeutsch changed to "NIST msp comment"

File: psi-ms.obo [hunk 1]
```diff
@@ -20081,6 +20081,42 @@ name: LTQ Orbitrap Velos Pro
...
+[Term]
+id: MS:1003101
+name: MaxQuant-DIA PEP
+def: "PSM evidence PEP probability from MAxQuant-DIA algorithm." [PSI:MS]
+is_a: MS:1001143 ! PSM-level search engine specific statistic
+
+[Term]
+id: MS:1003102
+name: NIST msp comment
+def: "NIST msp comment" [PSI:MS]
```
Comment (mwalzer) [inline psi-ms.obo:20117]: What is a NIST msp comment??

File: psi-ms.obo [hunk 2]
```diff
@@ -20081,6 +20081,42 @@ name: LTQ Orbitrap Velos Pro
...
+[Term]
+id: MS:1003099
+name: MaxQuant-DIA peptide PEP
+def: "Peptide probability from MaxQuant-DIA algorithm." [PSI:MS]
+is_a: MS:1002358 ! search engine specific peptide sequence-level identification statistic
+
+[Term]
+id: MS:1003100
+name: MaxQuant-DIA score
+def: "PSM evidence score from MAxQuant-DIA algorithm." [PSI:MS]
```
Comment (mwalzer) [inline psi-ms.obo:20105]: 'MAxQuant'

Comment (ypriverol) [reply psi-ms.obo]: Fixed.

File: psi-ms.obo (hunk 1 above)
Comment (ypriverol) [reply psi-ms.obo:20117]: In the NIST msp file format, one of the field of the spectrum is Comment. We need a Cvterm for it. See example here: https://raw.githubusercontent.com/HUPO-PSI/mzSpecLib/master/legacy-formats/msp-files/nist-bsa_consensus.msp

**Comment by ypriverol** [comment]
@edeutsch changes added.

**Pull request merged**